app/assets/javascripts/helpers/browser_notification.js

```diff
@@ -0,0 +1,33 @@
+Diaspora.BrowserNotification = {
+  requestPermission: function() {
+    if ("Notification" in window) {
+      if (Notification.permission !== "granted" && Notification.permission !== "denied") {
+        Notification.requestPermission();
+      }
+    } else {
+      /* eslint-disable no-console */
+      console.warn("Browser do not support notifications");
+      /* eslint-enable no-console */
```

Comment:
Yeah, I'd rather not have console logging. Believe it or not, there are browsers who don't even have the `console` object, so this would raise. Also, this is not a hard requirement that would break anything, so remove it.